### ISSUE TITLE
Uploader group

### ISSUE THREAD

**Issue by hevisko** [WWBN/AVideo#8760]
**Is your feature request related to a problem? Please describe.**
It seems (after checking the documentation and the installed test site) that Uploading is not a clear cut as a group setting (ie. to make a group for Video uploading and those members in that group have the permissions to upload)

In my case, I want people to join the site, they will be vetted by a non-technical person, but then I need a select group of users to be given the rights to upload videos. I can go into the detail settings and enable it for a user, but I'm looking for a clear cut method like just add to the uploaders group.

**Describe the solution you'd like**
Simple group membership to allow uploading rights

**Describe alternatives you've considered**
Painfull to enable it per uploader, but I'm hitting my head against the other settings that seems to allow it for all users instad of only a select group

**Additional context**
For me a "verified user" only has Viewing rights.
It must be a specific group addition to allow uploads, as we don't want (even just a channel for that user) poison the well with just any upload... and besides, the uploads will take resources away that we can't afford as is

**Comment by DanielnetoDotCom** [comment]
Currently "upload videos rights" is not in the user groups, but in the user configuration itself

**Comment by hevisko** [comment]
Yes, I noticed that, but want to make that a group type setting, not a user only setting.

Especially as the documentation/etc. isn't clear about the impact that the uploader setting has especially when you select `must have email verified`
